Auth/auth-common.ts

```diff
@@ -0,0 +1,10 @@
+import * as definition from "nativescript-azure-mobile-apps/user";
+
+export abstract class MobileServiceUser implements definition.MobileServiceUser {
+    
+    abstract login(provider: string): Promise<definition.MobileServiceUser>;
```

Comment:
I dont think the user needs a login method, as we already have this in the `client` file